ctit088
2024-06-03 16:45:51,120	ERROR services.py:1207 -- Failed to start the dashboard 
2024-06-03 16:45:51,232	ERROR services.py:1232 -- Error should be written to 'dashboard.log' or 'dashboard.err'. We are printing the last 20 lines for you. See 'https://docs.ray.io/en/master/ray-observability/ray-logging.html#logging-directory-structure' to find where the log file is.
2024-06-03 16:45:51,233	ERROR services.py:1242 -- Couldn't read dashboard.log file. Error: [Errno 2] No such file or directory: '/local/ray/session_2024-06-03_16-45-25_996557_3265374/logs/dashboard.log'. It means the dashboard is broken even before it initializes the logger (mostly dependency issues). Reading the dashboard.err file which contains stdout/stderr.
2024-06-03 16:45:51,233	ERROR services.py:1276 -- Failed to read dashboard.err file: cannot mmap an empty file. It is unexpected. Please report an issue to Ray github. https://github.com/ray-project/ray/issues
2024-06-03 16:45:25,909	INFO usage_lib.py:412 -- Usage stats collection is enabled. To disable this, add `--disable-usage-stats` to the command that starts the cluster, or run the following command: `ray disable-usage-stats` before starting the cluster. See https://docs.ray.io/en/master/cluster/usage-stats.html for more details.
2024-06-03 16:45:25,910	INFO scripts.py:722 -- Local node IP: 10.20.240.18
2024-06-03 16:45:58,185	SUCC scripts.py:759 -- --------------------
2024-06-03 16:45:58,186	SUCC scripts.py:760 -- Ray runtime started.
2024-06-03 16:45:58,186	SUCC scripts.py:761 -- --------------------
2024-06-03 16:45:58,186	INFO scripts.py:763 -- Next steps
2024-06-03 16:45:58,186	INFO scripts.py:766 -- To add another node to this Ray cluster, run
2024-06-03 16:45:58,186	INFO scripts.py:769 --   ray start --address='10.20.240.18:6379'
2024-06-03 16:45:58,186	INFO scripts.py:778 -- To connect to this Ray cluster:
2024-06-03 16:45:58,186	INFO scripts.py:780 -- import ray
2024-06-03 16:45:58,186	INFO scripts.py:781 -- ray.init()
2024-06-03 16:45:58,187	INFO scripts.py:812 -- To terminate the Ray runtime, run
2024-06-03 16:45:58,187	INFO scripts.py:813 --   ray stop
2024-06-03 16:45:58,188	INFO scripts.py:816 -- To view the status of the cluster, use
2024-06-03 16:45:58,188	INFO scripts.py:817 --   ray status
2024-06-03 16:50:24.888874: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 16:51:05.312265: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 16:52:33.806529: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 16:59:25.325749: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 16:59:25.382234: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 16:59:39.850940: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:00:44.400562: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:00:44.457874: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:01:04.443554: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:02:48.536018: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:02:48.593505: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:03:02.107601: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:04:16.764146: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:04:16.820613: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:04:30.247537: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:05:37.589212: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:05:37.646037: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:05:57.852295: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:07:02.980080: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:07:03.037212: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:07:15.795431: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:08:40.852684: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:08:40.918848: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:08:55.650578: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:10:04.137603: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:10:04.193657: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:10:14.593912: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:11:20.336235: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:11:20.393391: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:11:30.820698: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:12:42.049326: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:12:42.105622: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:12:49.297031: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:13:58.717921: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:13:58.774887: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:14:06.118196: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:15:05.327071: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:15:05.382914: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:15:26.157720: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:17:28.891743: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:17:28.948987: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:17:46.924696: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:19:00.605197: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:19:00.662906: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:19:11.845916: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:19:53.934886: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:19:53.994954: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:20:00.498975: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:20:35.339040: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-06-03 17:20:35.396323: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-06-03 17:20:46.699097: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/s2240084/conFEDential/src/main.py", line 17, in <module>
    from src.utils import batch_config
  File "/home/s2240084/conFEDential/src/utils/batch_config.py", line 7, in <module>
    from src.experiment import Config
  File "/home/s2240084/conFEDential/src/experiment/__init__.py", line 8, in <module>
    from .Attack import Attack
  File "/home/s2240084/conFEDential/src/experiment/Attack.py", line 6, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
srun: error: ctit088: task 0: Exited with exit code 1
2024-06-03 17:21:46,858	VINFO scripts.py:1068 -- Send termination request to `/home/s2240084/conFEDential/venv/bin/python -u /home/s2240084/conFEDential/venv/lib/python3.10/site-packages/ray/autoscaler/_private/monitor.py --logs-dir=/local/ray/session_2024-06-03_16-45-25_996557_3265374/logs --logging-rotate-bytes=536870912 --logging-rotate-backup-count=5 --gcs-address=10.20.240.18:6379 --monitor-ip=10.20.240.18` (via SIGTERM)
2024-06-03 17:21:46,858	VINFO scripts.py:1068 -- Send termination request to `/home/s2240084/conFEDential/venv/bin/python -u /home/s2240084/conFEDential/venv/lib/python3.10/site-packages/ray/_private/log_monitor.py --logs-dir=/local/ray/session_2024-06-03_16-45-25_996557_3265374/logs --gcs-address=10.20.240.18:6379 --logging-rotate-bytes=536870912 --logging-rotate-backup-count=5` (via SIGTERM)
2024-06-03 17:21:46,861	VINFO scripts.py:1068 -- Send termination request to `/home/s2240084/conFEDential/venv/bin/python -m ray.util.client.server --address=10.20.240.18:6379 --host=0.0.0.0 --port=10001 --mode=proxy --metrics-agent-port=46629` (via SIGTERM)
2024-06-03 17:21:46,871	VINFO scripts.py:1068 -- Send termination request to `/home/s2240084/conFEDential/venv/bin/python -u /home/s2240084/conFEDential/venv/lib/python3.10/site-packages/ray/_private/log_monitor.py --logs-dir=/local/ray/session_2024-06-03_16-45-25_996557_3265374/logs --gcs-address=10.20.240.18:6379 --logging-rotate-bytes=536870912 --logging-rotate-backup-count=5` (via SIGTERM)
2024-06-03 17:21:46,879	VINFO scripts.py:1068 -- Send termination request to `/home/s2240084/conFEDential/venv/bin/python /home/s2240084/conFEDential/venv/lib/python3.10/site-packages/ray/dashboard/dashboard.py --host=127.0.0.1 --port=8265 --port-retries=0 --temp-dir=/local/ray --log-dir=/local/ray/session_2024-06-03_16-45-25_996557_3265374/logs --session-dir=/local/ray/session_2024-06-03_16-45-25_996557_3265374 --logging-rotate-bytes=536870912 --logging-rotate-backup-count=5 --gcs-address=10.20.240.18:6379 --node-ip-address=10.20.240.18 --minimal` (via SIGTERM)
2024-06-03 17:21:47,149	INFO scripts.py:1098 -- 1/4 stopped.2024-06-03 17:21:47,149	INFO scripts.py:1098 -- 2/4 stopped.2024-06-03 17:21:47,149	INFO scripts.py:1098 -- 3/4 stopped.2024-06-03 17:21:47,684	INFO scripts.py:1098 -- 4/4 stopped.2024-06-03 17:21:47,808	VINFO scripts.py:1068 -- Send termination request to `/home/s2240084/conFEDential/venv/lib/python3.10/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/local/ray/session_2024-06-03_16-45-25_996557_3265374/logs --config_list=eyJvYmplY3Rfc3BpbGxpbmdfY29uZmlnIjogIntcInR5cGVcIjogXCJmaWxlc3lzdGVtXCIsIFwicGFyYW1zXCI6IHtcImRpcmVjdG9yeV9wYXRoXCI6IFwiL2xvY2FsL3JheS9zZXNzaW9uXzIwMjQtMDYtMDNfMTYtNDUtMjVfOTk2NTU3XzMyNjUzNzRcIn19IiwgImlzX2V4dGVybmFsX3N0b3JhZ2VfdHlwZV9mcyI6IHRydWV9 --gcs_server_port=6379 --metrics-agent-port=46629 --node-ip-address=10.20.240.18 --session-name=session_2024-06-03_16-45-25_996557_3265374` (via SIGTERM)
2024-06-03 17:21:48,624	INFO scripts.py:1098 -- 1/1 stopped.2024-06-03 17:21:48,625	SUCC scripts.py:1142 -- Stopped all 5 Ray processes.
